packages/explorer/README.md

```diff
@@ -1,32 +1,38 @@
 # World Explorer
 
 World Explorer is a GUI tool designed for visually exploring and manipulating the state of worlds.
+For the full information about it, [see the official docs](http://mud.dev/world-explorer).
 
 ## Getting started
 
-1. **Install the package**
+### Prerequisites
 
-   ```sh
-   pnpm add @latticexyz/explorer
-   ```
+World Explorer is not intended to be used alone, and includes the following prerequisites to be set up and running:
 
-2. **Start a local development chain**
+- MUD project
+- Development chain (e.g. Hardhat, Foundry)
```

Comment:
we have a fairly strong coupling to foundry so not sure it's worth mentioning hardhat?

Reply:
true, changed to reference `anvil` specifically